RoosterPlanner.Api/Models/ProjectTaskViewModel.cs

```diff
@@ -0,0 +1,47 @@
+﻿using System;
+using RoosterPlanner.Models;
+
+namespace RoosterPlanner.Api.Models
+{
+    public class ProjectTaskViewModel
+    {
+        public Guid Id { get; set; }
+
+        public Guid ProjectId { get; set; }
+        public Guid TaskId { get; set; }
+
+        public static ProjectTaskViewModel CreateVm(ProjectTask projectTask)
+        {
+            if (projectTask != null)
+            {
+                ProjectTaskViewModel vm = new ProjectTaskViewModel()
+                {
+                    Id = projectTask.Id,
+                    ProjectId = projectTask.ProjectId,
+                };
+                if (projectTask.TaskId != null)
```

Comment:
Task is een Guid. Deze kan niet null zijn, hoogstend Guid.Empty. (Guid is een Value Type die krijgt een default waarde als deze niet Nullable is. Alleen Ref Typen, zoals string, object zijn default null)

Reply:
Deze was ik toevallig ook in de refactor branch tegen gekomen. Ik neem hem hier ook mee